charts/plot-navigator/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: plot-navigator
+spec:
+  selector:
+    app: plot-navigator
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: {{ .Values.port | default 8080 }}
```

Comment:
`port` appears to be missing from `Values.yaml`. You have a default so it probably all works, but I think this is a little less legible from the outside than including `port` with a default.

Reply:
I don't have a good reason for this to be configurable, so I just removed it.